terrautils/metadata/lemnatec.py

```diff
@@ -1,24 +1,32 @@
-'''
-Create a cleaned/curated metadata object based on the LemnaTec fixed metadata
-Some metadata used for search
-Some metadata used for extractors
+"""
+Given a JSON object containing the original LemnaTec metadata, convert
+field names and values to standardized formats required for TERRA-REF.
```

Comment:
Probably need to revisit a few points from the earlier metadata standards review.  SensorML defines an abstract model, but the actual implementation and tooling is all centered around XML.  The MMI group did define some RDF contexts for linked data for some of the SensorML concepts, but these haven't been standardized (i.e., blessed by OGC).  

What you seem to be proposing is that we re-implement the LemnaTec sensor metadata in SensorML-compliant format, which seems well out of scope. In my mind, if the sensor manufacturer provides SensorML compliant metadata, we should work with it, but I don't see the case for implementing SensorML specifications for every sensor for every project in TERRA.  

That said, the LemnaTec metadata is riddled with errors -- typos, inconsistencies, case issues, etc, that we can correct.  Additionally,  spaces in keys in JSON objects can cause issues with some parsers.  

For the November release and the work proposed in terra-utils, I think we should focus on what is necessary for the v1 release.  If there are changes that you think are necessary for v1, understanding that to-date extractors have been working with the Lemnatec variable metadata as-is, please let me know.